content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -29,137 +29,135 @@ further_reading:
     text: "Examples of Custom Instrumentation"
 ---
 
-## Compatibility Requirements
+## Compatibility requirements
 
-The .NET Tracer supports automatic instrumentation on .NET 5, .NET Core 3.1, and .NET Core 2.1. It also supports [.NET Framework][1]. For a full list of supported libraries, visit the [Compatibility Requirements][2] page.
+The .NET Tracer supports instrumentation on:
+  - .NET 5
+  - .NET Core 3.1
+  - .NET Core 2.1. 
 
-## Getting started
+For a full list of supported libraries, visit the [Compatibility Requirements][1] page.
 
-**Note**: The .NET Tracer supports all .NET-based languages (C#, F#, Visual Basic, etc).
+## Automatic instrumentation
 
-## Automatic Instrumentation
-
-Automatic instrumentation can collect performance data about your application with zero code changes and minimal configuration. The .NET Tracer automatically instruments all [supported libraries][2] out of the box.
-
-Automatic instrumentation captures:
-
-- Execution time of instrumented calls
-- Relevant trace data, such as URL and status response codes for web requests or SQL queries for database access
-- Unhandled exceptions, including stacktraces if available
-- A total count of traces (e.g. web requests) flowing through the system
+<div class="alert alert-warning"> 
+  <strong>Note:</strong>  If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.
+</div>
 
 ### Installation
 
+By default, tracing is enabled in your `datadog.yaml` file under `apm_enabled: true` and the Agent listens for trace traffic at `localhost:8126`.
```

Comment:
I think having this as the first line is confusing. It refers to Agent settings, but this section is about Tracer installation. There is not `datadog.yaml` in the Tracer. Did we intentionally remove the section "Configure the Datadog Agent for APM" below, @jaycdave88?

Reply:
@lucaspimentel we did in order to focus on a more steps rather then large texts. Kari and I chatted on this - I was for removing since it doesnt exist today in our public docs for .NET Framework.

Reply:
https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-framework#configure-the-datadog-agent-for-apm

Reply:
Ah, thank you for the flag. 

Yes - lets just add that same section to our .NET Core docs. @kayayarai